server/game/cards/11.6-DitD/CunningDiplomat.js

```diff
@@ -0,0 +1,27 @@
+const DrawCard = require('../../drawcard.js');
+
+class CunningDiplomat extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardEntersPlay: event => (
+                    event.card === this &&
+                    this.game.isDuringChallenge()
+                )
+            },
+            target: {
+                cardCondition: card => card.getType() === 'character' && card.isParticipating()
+            },
+            handler: context => {
+                context.target.controller.standCard(context.target);
+                this.game.currentChallenge.removeFromChallenge(context.target);
+
+                this.game.addMessage('{0} uses {1} to remove {2} from the challenge', context.player, this, context.target);
```

Comment:
The message should probably mention that the targeted character is stood, e.g. "to stand and remove {2}"